contracts/Turnstone.vy

```diff
@@ -0,0 +1,142 @@
+# @version 0.3.3
+"""
+@title Turnstone-EVM
+@author Volume.Finance
+"""
+
+MAX_VALIDATORS: constant(uint256) = 128
+
+POWER_THRESHOLD: constant(uint256) = 2_863_311_530 # 2/3 of 2^32, Validator powers will be normalized to sum to 2 ^ 32 in every valset update.
+TURNSTONE_ID: immutable(bytes32)
+
+struct Valset:
+    validators: DynArray[address, MAX_VALIDATORS] # Validator addresses
+    powers: DynArray[uint256, MAX_VALIDATORS] # Powers of given validators, in the same order as validators array
+    valset_id: uint256 # nonce of this validator set
+
+struct Signature:
+    v: uint256
+    r: uint256
+    s: uint256
```

Comment:
It's not clear what I have to send for the signature. What are v, r, s?

Reply:
Does this help you?

Reply:
It does. Can I send you the byte array that I am getting when doing the keccak 256 crypto signing and you extract those values?